public/app/core/components/scroll/scroll.ts

```diff
@@ -31,13 +60,25 @@ export function geminiScrollbar() {
         scope
       );
 
+      // force updating dashboard width
+      appEvents.on('toggle-sidemenu', forceUpdate);
```

Comment:
this could be a memory leak, no unsubscribe  (or scope passed as last argument to unsubscribe on scope destory)

Reply:
Ah, yea, missed that.